File: BloodFlow360.React/src/types/setting.ts
export interface Setting {
  id: string;
  settingKey: string;
  settingValue: string;
  description: string;
  isEncrypted: boolean;
}

export interface CreateSetting {
  settingKey: string;
  settingValue: string;
  description: string;
  isEncrypted: boolean;
}

export interface UpdateSetting {
  settingValue: string;
  description: string;
  isEncrypted: boolean;
}

File: BloodFlow360.Domain/Entities/Setting.cs
using BloodFlow360.Domain.Common;

namespace BloodFlow360.Domain.Entities;

public class Setting : BaseEntity
{
    public string SettingKey { get; set; } = string.Empty;

    public string SettingValue { get; set; } = string.Empty;

    public string Description { get; set; } = string.Empty;

    public bool IsEncrypted { get; set; }
}